### PR TITLE
refactor ColorSelector

### PR DESCRIPTION
**Pull request by jujuzitroni**
I've changed my color selector from a html selector to an ul, so that I can style the dropdown the way I want.

to test go to
https://plants-need-input-irjitwngq5sf.herokuapp.com/create-habit
click on color dot to open dropdown.
select a color and dropdown closes.

this branch is behind, which is why the page looks wonky.

### CODE REVIEW

**Review by Niklas-Schaefers — APPROVED** [review]
lgtm 🚀

File: src/app/config.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export const colors = [
```
Comment (lmachens) [inline src/app/config.ts:1]:
```suggestion
export const colors: string[] = [
```

File: src/app/components/ColorSelector/ColorSelector.tsx [hunk 2]
```diff
@@ -1,26 +1,49 @@
...
+
   return (
-    <label>
-      <span>choose color</span>
-      <select
-        name="colorSelector"
-        onChange={(event) => onSelectColor(event.target.value)}
+    <div className={styles.colorSelector}>
+      <h2>choose color</h2>
+      <div
```
Comment (lmachens) [inline src/app/components/ColorSelector/ColorSelector.tsx:19]:
```suggestion
      <button
```
I would change it to a button to have full browser support. You can style the button like a div.

File: src/app/components/ColorSelector/ColorSelector.tsx [hunk 3]
```diff
@@ -1,26 +1,49 @@
...
+        className={styles.colorSelector__selected}
```
Comment (lmachens) [inline src/app/components/ColorSelector/ColorSelector.tsx:20]:
```suggestion
        className={styles.colorSelector__select}
```

File: src/app/components/ColorSelector/ColorSelector.tsx [hunk 4]
```diff
@@ -1,26 +1,49 @@
...
+          style={{ backgroundColor: selectedColor }}
+        />
+      </div>
+      {isListOpen && (
+        <ul className={styles.colorSelector__dropdown}>
+          {colors.map((color) => (
+            <li key={color}>
+              <div
+                className={styles.colorSelector__dots}
+                style={{ backgroundColor: color }}
```
Comment (lmachens) [inline src/app/components/ColorSelector/ColorSelector.tsx:36]: You could remove the `div` and style the `li`

File: src/app/components/ColorSelector/ColorSelector.stories.tsx [hunk 5]
```diff
@@ -6,6 +6,12 @@ export default {
...
+let selectedColor = 'blue';
+
+function changeColor(color: string) {
+  selectedColor = color;
+}
+
 export const Selector = (): JSX.Element => (
-  <ColorSelector onSelectColor={console.log} />
+  <ColorSelector selectedColor={selectedColor} onSelectColor={changeColor} />
 );
```
Comment (lmachens) [inline src/app/components/ColorSelector/ColorSelector.stories.tsx:17]: This won't work. If you like to change the selected color, add a state:

```tsx
export const Selector = (): JSX.Element => {
  const [selectedColor, setSelectedColor] = useState('blue');
  return <ColorSelector selectedColor={selectedColor} onSelectColor={setSelectedColor} />;
}
```

**Review by lmachens — CHANGES_REQUESTED** [review]
gj!

**Pull request merged**